locale/flarum-tags.yml

```diff
@@ -33,6 +33,7 @@ flarum-tags:
     permissions:
       restrict_by_tag_heading: Restrict by Tag
       tag_discussions_label: Tag discussions
+      allow_edit_tags_label: Allow edit tags
```

Comment:
Can you call this "Allow editing tags" please? Or any other suggestions, @tobscure? You're the native speaker. ;)